.teamcity/Gradle_Check/configurations/IndividualPerformanceScenarioWorkers.kt

```diff
@@ -20,24 +20,35 @@ class IndividualPerformanceScenarioWorkers(model: CIBuildModel, os: Os = Os.linu
     applyPerformanceTestSettings(os = os, timeout = 420)
     artifactRules = individualPerformanceTestArtifactRules
 
+    if (os == Os.windows) {
+        // to avoid pathname too long error
+        vcs {
+            checkoutDir = "C:\\gradle-perf"
+        }
+    }
+
     params {
         param("baselines", "defaults")
         param("templates", "")
-        param("channel", "commits")
+        param("channel", if (os == Os.linux) "commits" else "commits-windows")
         param("checks", "all")
         param("runs", "defaults")
         param("warmups", "defaults")
         param("scenario", "")
 
-        param("env.ANDROID_HOME", "/opt/android/sdk")
-        param("env.PATH", "%env.PATH%:/opt/swift/4.2.3/usr/bin")
+        param("env.ANDROID_HOME", os.androidHome)
+        if (os == Os.linux) {
+            param("env.PATH", "%env.PATH%:/opt/swift/4.2.3/usr/bin")
```

Comment:
What about macOS?

Reply:
I think this `if` should be a `switch` instead.

Reply:
done.